docs/content/manual/harvester-rancher/68-Fully-airgapped-rancher-integrate-harvester-no-proxy.md

```diff
@@ -0,0 +1,511 @@
+  wget https://github.com/k3s-io/k3s/releases/download/v1.23.4%2Bk3s1/k3s
+  chmod +x k3s
+  
+  sudo cp k3s /usr/local/bin/
+  sudo chown $USER /usr/local/bin/k3s
+  
+  curl https://get.k3s.io/ -o install.sh
+  chmod +x install.sh
+  ```
+2. Download rancher v2.6,4-rc13 helm
+  ```
+  helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
+  helm fetch rancher-latest/rancher --version=v2.6.4-rc13
+  ```
+3. Download cert-manager crds.
+  ```
+  mkdir cert-manager
+  curl -L -o cert-manager/cert-manager-crd.yaml https://github.com/jetstack/cert-manager/releases/download/v1.7.1/cert-manager.crds.yaml
+  ```
+
+4. Install k9s (option)
+  ```
+  curl -kL https://github.com/derailed/k9s/releases/download/v0.25.18/k9s_Linux_x86_64.tar.gz > k9s.tar.gz
+  tar -zxvf k9s.tar.gz
+  mv k9s /usr/local/bin/
+  ```
+
+5. Cut Network -> Remove the `vagrant-libvirt` network device from virtual machine, make it air gapped
+6. Load Rancher images to private registry. 
+  ```
+  ./rancher-load-images.sh --image-list ./rancher-images.txt --registry myregistry.local:5000
+  ```
+  - This step may take 50 minutes depends on total image size
+
+#### Phase 5: Install K3s and Rancher 
+1. Move k3s images files.
+  ```
+  sudo mkdir -p /var/lib/rancher/k3s/agent/images/
+  sudo cp ./k3s-airgap-images-amd64.tar /var/lib/rancher/k3s/agent/images/
+  ```
+2. Install k3s
+  ```
+  INSTALL_K3S_SKIP_DOWNLOAD=true ./install.sh
+  ```
+3. Add `registries.yaml`  to `/etc/rancher/k3s/`
+  ```
+  # vim /etc/rancher/k3s/registries.yaml
+  mirrors:
+    docker.io:
+      endpoint:
+        - "https://myregistry.local:5000/"
+  configs:
+    "myregistry.local:5000":
+      tls:
+        insecure_skip_verify: true
+  ```
+4. Restart k3s
+  ```
+  sudo systemctl restart k3s.service
+  ```
+5. Copy kubeconfig to ~/.kube
+  ```
+  mkdir ~/.kube
+  sudo cp /etc/rancher/k3s/k3s.yaml ~/.kube/config
+  sudo chown $USER ~/.kube/config
+  export KUBECONFIG=~/.kube/config
+  ```
+6. Generate cert-manager YAML files.
+  ```
+  helm template cert-manager ./cert-manager-v1.7.1.tgz --output-dir . \
+      --namespace cert-manager \
+      --set image.repository=myregistry.local:5000/quay.io/jetstack/cert-manager-controller \
+      --set webhook.image.repository=myregistry.local:5000/quay.io/jetstack/cert-manager-webhook \
+      --set cainjector.image.repository=myregistry.local:5000/quay.io/jetstack/cert-manager-cainjector \
+      --set startupapicheck.image.repository=myregistry.local:5000/quay.io/jetstack/cert-manager-ctl
+  ```
+7. Install cert-manager.
+  ```
+  kubectl create namespace cert-manager
+  kubectl apply -f cert-manager/cert-manager-crd.yaml
+  kubectl apply -R -f ./cert-manager
+  ```
+8. Create `create-self-signed-cert.sh`, copy the content in the [signed certificate guide ](http://docs.rancher.cn/docs/rancher2.5/installation/resources/advanced/self-signed-ssl/_index/#4-%E5%A6%82%E4%BD%95%E7%94%9F%E6%88%90%E8%87%AA%E7%AD%BE%E5%90%8D%E8%AF%81%E4%B9%A6)
```

Comment:
This isn't available in the English docs. You can use the translate function, but it might be worth noting in the docs, and maybe we could request that they add it in the English docs.